site/testData/mockData.ts

```diff
@@ -877,6 +877,25 @@ export const mockSessionWithOrgDetail: SessionWithOrgDetail = {
     mode: defaultModes,
     isOperatorUser: false,
     operatorOrgId: null,
+    showUnderground: false,
+};
+
+export const mockSessionWithOrgDetailTFL: SessionWithOrgDetail = {
```

Comment:
```suggestion
export const mockSessionWithOrgDetailAndShowUndergroundTrue: SessionWithOrgDetail = {
```

I would probably rename and remove the reference to TfL - although we know it is for TfL we haven't referenced a specific org anywhere else in the code base :)

Reply:
done